MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -16,8 +16,14 @@ public partial class GraphicsDevice : IDisposable
         private bool _isDisposed;
 
         private BlendState _blendState;
-        private BlendState _actualBlendState;
-        private DepthStencilState _depthStencilState = DepthStencilState.Default;
+        private BlendState _actualBlendState;
```

Comment:
I was wondering if anyone (else) would notice that :-) Yeah, I planned to move those, just wasn't sure which PR to fit that into.